### DISCUSSION TITLE
烂尾的窟窿查100个贪官填补是否可行

### DISCUSSION THREAD

**Discussion by ppwq2022** [WeNeedHome/SummaryOfLoanSuspension#435] [General]
RT，一个市或者县查一个

**Comment by kisdaray** [comment]
10个就够了100个用不完

**Reply by woyewangle** [reply]
之前不是查处了河南副省长刘学周吗？ 一个人就是1060亿了.。10个才1W亿 得要20个副省长

**Comment by baoang** [comment]
从统计学上说，这取决于这100个抽样出来的贪官是否具有代表性，如果具有，那么可行与否取决于每个平均的可没收家产有多少，以及100个当中“假阳性”的比率。

**Comment by lettermessenger** [comment]
那说的这100个贪官包不包括去调查监管的那个官？

**Comment by markKscode** [comment]
查100个大官？那这中共内部怕是要闹翻天了。

**Comment by Siryedu** [comment]
你让他查他能查嘛？没那个实力知道吧？

**Comment by JayChou55** [comment]
唐山都那B样了 信访举报队伍都排到马路上了 现在有下文吗，小唐山都不查 还查大的？

**Reply by Siryedu** [reply]
自罚三杯得了，你这样纠缠有意思吗？

**Comment by k624035508** [comment]
和珅打倒   嘉庆吃饱   贪官倒了跟你韭菜有什么关系

**Comment by zhw173** [comment]
贪的越多官越大啊，这不是相当于让他们自己出钱堵窟窿

**Comment by volans19** [comment]
一个就够了，一个破银行的副行长都能做200万亿的帐，5个都多了

**Comment by shawn-yee** [comment]
赖小民*100

**Comment by clleo888** [comment]
查了有用吗? 贪官的钱只是去了另一个贪官手裡,那次是归还给被害人的?

**Comment by Accelerator6666** [comment]
那些人的钱全都去了瑞士银行咯。。。追的回来？
